Formula/vault.rb

```diff
@@ -33,6 +35,12 @@ class Vault < Formula
   def install
     ENV["GOPATH"] = buildpath
 
+    if build.with? "dynamic"
+      target = "dev-dynamic"
+    else
+      target = "dev"
+    end
```

Comment:
`target = build.with?("dynamic) ? "dev-dynamic" : "dev"` and stick it immediately above `system "make", target`